spec/plugins/url_plugin_spec.rb

```diff
@@ -9,6 +9,12 @@
 #testing a failing url
 failing_url  = { url: 'http://www.google.com/fail/'}
 
+multiple_retry_monitor = {
+  url: 'http://www.google.com/fail/',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.